nvmbugs/002_simple_symbolic/simple_symbolic.c

```diff
@@ -0,0 +1,30 @@
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
This seems like a duplicate of `002_SymbolicOffsetTest`

Reply:
hm, oops, might be. I only looked at the other 002

Reply:
I remember, it is a duplicate, but not using mmap, just using klee_mark_persistent on a stack buffer.